targets/XPlatCppSdk/templates/PlayFab_Api.cpp.ejs

```diff
@@ -32,7 +41,16 @@ namespace PlayFab
 <%- getRequestActions("        ", apiCall) %>
         IPlayFabHttp& http = IPlayFabHttp::Get();
         const auto requestJson = request.ToJson();
-        http.AddRequest("<%- apiCall.url %>", <%- getAuthParams(apiCall) %>, requestJson, On<%- apiCall.name %>Result, SharedVoidPointer((callback == nullptr) ? nullptr : new ProcessApiCallback<<%- apiCall.result %>>(callback)), errorCallback, customData);
+
+        Json::FastWriter writer;
+        std::string jsonAsString = writer.write(requestJson);
+
+        std::unordered_map<std::string, std::string> headers;
+        headers.emplace(<%- getAuthParams(apiCall) %>);
```

Comment:
This is really great. We set the auth at the very beginning, and never have to think about them again.
Every bit of auth processing AFTER this point is redundant.

Reply:
(What happens if auth is null/null, such as login calls?)

Reply:
Normal thing will happen: auth is passed as two empty strings and acts in the same manner as it did before.